tools/cd_scripts/e2e_test.sh

```diff
@@ -71,9 +71,8 @@ then
     #install git
     sudo apt install -y git
 
-    # Downloading one or more large file(s), which would run significantly faster if
-    # you enabled sliced object downloads. This feature is enabled by default but
-    # requires that compiled crcmod be installed.
+    # Downloading composite object requires integrity checking with CRC32c in gsutil.
```

Comment:
All these utilities do already present on other machines?

Reply:
On some machines
```
gsutil version -l
compiled crcmod: False
```

and on some machines it's by default true